Tools/Scripts/libraries/webkitscmpy/webkitscmpy/local/git.py

```diff
@@ -1032,8 +1032,27 @@ def checkout(self, argument, prune=None):
                 cwd=self.root_path,
             ).returncode else self.commit()
 
+        match = self.dev_branches.match(argument)
         branch_remote = self.remote_for(argument)
-        if branch_remote:
+
+        # if branch has a remote and is not a dev branch
+        if not match and branch_remote:
+            path = '{}/{}'.format(branch_remote, argument)
+            sys.stderr.write("Resetting branch {} to remote {}.\n Checkout will erase all un-pushed changes on {}.\n".format(argument, path, argument))
```

Comment:
We generally reserve `stderr` for something going wrong, I don't think that's the case here. This seems like `log.info`, or maybe even `log.debug`.

As a general statement, `print` is reserved for required output of scripts (like `git-webkit pr` outputting the URL of the posted PR). `sys.stderr` is reserved for errors that we expect users to react to. `log.*` is for everything else (and notably, controlled by `-q` and `-v`, which isn't the case for `print` and `sys.stderr`)